src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,7 @@
+@function width-size($cell, $gap, $pad) {
```

Comment:
It's recommended to use a more descriptive function name, such as 'calendar-width-size'.